playwright/tests/orders.spec.ts

```diff
@@ -342,46 +342,58 @@ for (const refund of orderRefunds) {
     await expect(ordersPage.orderRefundList).not.toContainText(refund.status);
   });
 }
-test(`TC: SALEOR_215 Inline discount is applied in a draft order @draft @discounts @e2e`, async page => {
+
+test(`TC: SALEOR_215 Inline discount is applied in a draft order @draft @discounts @e2e`, async () => {
   test.slow();
 
+  const calculateDiscountedPrice = (
+    undiscountedPrice: number,
+    discountPercentage: number,
+  ): number => {
+    return undiscountedPrice - (undiscountedPrice * discountPercentage) / 100;
+  };
```

Comment:
This is dangerous: you just create a code to test the code. It implements the way discount is being calculated, meaning the test now is aware of it while it should not. Everywhere when possible make assertions on real values, because the incoming data is always known (you know what prices given product has because it's test data)

Non, blocking for now, but please keep in mind that.